### ISSUE TITLE
Explicit support for "no-project" selected

### ISSUE THREAD

**Issue by jldec** [opral/inlang-sherlock#18]
## Context

`loadProject()` should always return a `project` api, with any errors in project.errors. Since this is hard to guarantee, errors thrown by loadProject() are already partly handled in sherlock. but there are still several code paths which assume that state().project is not undefined.
e.g. [here](https://github.com/opral/monorepo/blob/main/inlang/source-code/ide-extension/src/main.ts#L134) and [here](https://github.com/opral/monorepo/blob/main/inlang/source-code/ide-extension/src/main.ts#L152) and [bug here](https://github.com/opral/monorepo/blob/main/inlang/source-code/ide-extension/src/utilities/errors/errors.ts#L38) 

## Proposal

It would be nice for users to be able to select "no project" e.g. if they are working on a non-i18n package in a monorepo, and don't want to pay for sherlock resource consumption.

The option would also be helpful to disable the default project message loading for repos with one really big project (1000's of messages) which (currently) slows down vs code and probably results in users uninstalling sherlock.

**Comment by felixhaeberle** [comment]
You are right 👍 With the introduction of the project tab, there is the case of "No project could can be found" and therefore we should also handle this explicitly in the extension.

**Comment by felixhaeberle** [comment]
with [SHERL-27](https://linear.app/opral/issue/SHERL-27/integrate-createnewproject-into-getting-started) getting shipped, this is going to improve, but there are still some code path which can be improved.

**Comment by samuelstroschein** [comment]
i think this has been implemented. re-open if not
